src/style.css

```diff
@@ -1 +1,34 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 22px;
+  display: inline-block;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16;
+  height: 16;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars:not(.stars--5) {
+  margin-bottom: 4px;
+}
+
+.stars__star:nth-last-child(n + 2) {
```

Comment:
`.stars__star:not(:last-child)` should also work but it looks a bit more understandable

FYI